### PR TITLE
[master] enable scilla unit tests in travisCI

### PR DESCRIPTION
**Pull request by Revolution1**
## Backward Compatibility

- [ ] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,51 +14,55 @@ addons:
...
+      - python3-dev
+      - libsecp256k1-dev
 
 install:
   - wget https://github.com/Kitware/CMake/releases/download/v3.19.3/cmake-3.19.3-Linux-x86_64.sh
   - mkdir -p "${HOME}"/.local
   - bash ./cmake-3.19.3-Linux-x86_64.sh --skip-license --prefix="${HOME}"/.local/
   - cmake --version
   - rm cmake-3.19.3-Linux-x86_64.sh
+  - docker run --rm -it -v /scilla:/root/scilla zilliqa/scilla:v0.9.1 cp -r /scilla /root/
```
Comment (KaustubhShamshery) [inline .travis.yml:60]: should make scilla latest?

Comment (ansnunez) [reply .travis.yml:60]: I don't think we build latest image for Scilla, only tags

**Review by vaivaswatha — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (KaustubhShamshery) [reply .travis.yml:60]: Okay

File: scripts/ci_build.sh [hunk 2]
```diff
@@ -69,7 +69,7 @@ then
     # The target Zilliqa_coverage already includes "ctest" command, see cmake/CodeCoverage.cmake
     cmake --build ${dir} --target Zilliqa_coverage || exit 1
 else
-    cd build && ctest --output-on-failure -j${n_parallel} || exit 1
+    cd build && ctest --output-on-failure -j1 || exit 1
```
Comment (KaustubhShamshery) [inline scripts/ci_build.sh:72]: can replace this too?

File: CMakeLists.txt [hunk 3]
```diff
@@ -319,7 +319,7 @@ if (ENABLE_COVERAGE AND CMAKE_COMPILER_IS_GNUCXX)
     link_libraries(--coverage)
     add_custom_target(ctest COMMAND ${CMAKE_CTEST_COMMAND})
     # TODO: remove the hardcoded number in -j option
-    setup_target_for_coverage(${PROJECT_NAME}_coverage ctest coverage "-j2;--output-on-failure")
+    setup_target_for_coverage(${PROJECT_NAME}_coverage ctest coverage "-j1;--output-on-failure")
```
[the remark this answers is not in the document]
Comment (ansnunez) [reply CMakeLists.txt:322]: Hmm I think multiple tests calling scilla won't work. For example, the pkill triggered at the end of one unit test for scilla cleanup might conflict with another unit test calling scilla

File: scripts/ci_build.sh (hunk 2 above)
Comment (ansnunez) [reply scripts/ci_build.sh:72]: Same reason as previous comment

**Pull request merged**